data/857/657/69/85765769.geojson

```diff
@@ -88,13 +115,15 @@
     "wof:lang":[
         "spa"
     ],
-    "wof:lastmodified":1582381931,
+    "wof:lastmodified":1705624223,
     "wof:name":"Ujarraz",
     "wof:parent_id":-1,
     "wof:placetype":"neighbourhood",
     "wof:repo":"whosonfirst-data-admin-cr",
     "wof:superseded_by":[],
-    "wof:supersedes":[],
+    "wof:supersedes":[
+        1126084379
```

Comment:
This `1126084379` record should be included in the diff but isn't – it should have been modified to way `"wof:superseded_by":[ 85765769 ]`. I wonder if `"wof:id"` is oddly missing from this record, so it failed because of that?

Reply:
Ah I think this is a case where we're crossing repos. https://spelunker.whosonfirst.org/id/1126084379/

This record lives in the `admin-xx` repo. I've confirmed the xx repo would get an update in this case, so I think we're ok here.